lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -62,13 +62,35 @@
     aliases:
       - incremental
 
+  enable_ipv6:
+    version_added: 2.8
+    description:
+      - Enable IPv6 networking.
+    type: bool
+    default: null
+    required: false
+
   ipam_driver:
     description:
       - Specify an IPAM driver.
 
   ipam_options:
     description:
       - Dictionary of IPAM options.
+    deprecated:
```

Comment:
Please remove this, and add an item with the deprecated explanation to `description`. Also, please use correct casing for `IPAM`.